neqo-http3/src/features/extended_connect/webtransport_streams.rs

```diff
@@ -91,6 +92,9 @@ pub(crate) struct WebTransportSendStream {
     events: Box<dyn SendStreamEvents>,
     session: Rc<RefCell<WebTransportSession>>,
     session_id: StreamId,
+    bytes_written: u64,
+    bytes_sent: u64,
+    bytes_non_app_data: u64,
```

Comment:
We should instead track everything necessary at the transport level.  This is a generic capability that would be useful for many other purposes.  The only correction you need specific to web transport is the removal of any stream header.  We can correct for with a simple calculation; see below.

Reply:
Will do.